src/technologies/f.json

```diff
@@ -446,16 +446,6 @@
     "scriptSrc": "cdn\\.usefathom\\.com/",
     "website": "https://usefathom.com"
   },
-  "Fbits": {
-    "cats": [
-      6
-    ],
-    "icon": "Fbits.png",
```

Comment:
This file needed to be deleted.

I've opened a new PR for this at #19 since I'm not able to edit this branch.